satpy/tests/reader_tests/test_olci_nc.py

```diff
@@ -241,7 +296,7 @@ def test_chl_nn(self, mocked_dataset):
         assert res.values[-1, -1] == 1e29
 
 
-class TestBitFlags(unittest.TestCase):
+class TestL2BitFlags(unittest.TestCase):
```

Comment:
This class is only a single function and has no setup. Feel free to extract the single method into a function and remove the class. This is also not using anything specific to unittest so it doesn't need to be a TestCase unittest anymore (but is being removed regardless).

Reply:
Sure, I will simplify the test from a class to a single function.